scripts/prepare-msm.sh

```diff
@@ -27,39 +30,9 @@ while [ -h "$SOURCE" ] ; do
 done
 DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
-# Determine which user is running this script
-setup_user=$USER
-setup_group=$( id -gn $USER )
-
-function found_exe() {
-    hash "$1" 2>/dev/null
-}
-
-if found_exe sudo ; then
-    # The main checks happen in dev_setup.sh, don't error here if we don't have sudo
-    SUDO=sudo
-fi
-
-# change ownership of ${mycroft_root_dir} to ${setup_user } recursively
-function change_ownership {
-    echo "Changing ownership of" ${mycroft_root_dir} "to user:" ${setup_user} "with group:" ${setup_group}
-    $SUDO chown -Rf ${setup_user}:${setup_group} ${mycroft_root_dir}
-}
-
-
 if [[ ${CI} != true ]] ; then
```

Comment:
I wonder, is this condition still required?
No matter if we're running on CI or not, making sure the skills directory exists doesn't hurt.

Getting rid of this means we can get rid of the sourcing and changing directories above this, and _that_ means we can run this script on a pure POSIX shell.

Reply:
Actually never mind, this script isn't needed at all anymore. The new skill directories don't require root rights and will be made by the pyxdg dep if they don't exist yet, so no need preparing anything. I'll just remove the script.